src/jvm/test/resources/expected_logs/clocks_and_exceptions.txt

```diff
@@ -0,0 +1,53 @@
+= Invalid execution results =
+Init part:
+[operation2(): void, operation1(): void, operation1(): void, operation2(): void, operation2(): void]
+Parallel part:
+| operation1(): void       | operation2(): IllegalStateException #1 [1,-] |
+| operation1(): void [1,-] | operation1(): void                     [1,1] |
+Post part:
+[operation1(): void, operation1(): void, operation1(): void, operation2(): void, operation1(): void]
+
+---
+values in "[..]" brackets indicate the number of completed operations 
+in each of the parallel threads seen at the beginning of the current operation
+---
+Number after exception name is used to match exception with it's 
+stacktrace in the exceptions section below the interleaving section
```

Comment:
Let's try to make the message shorter. Suggestion: "The number next to an exception name helps you find its stack trace provided after the interleaving section."